CONTRIBUTING.md

```diff
@@ -87,3 +89,343 @@ generated using the pre-commit hooks for
+        tag: v1.6.0.dev7
+    pluginsSettings:
+      image:
+        repository: us-central1-docker.pkg.dev/computer-vision-team/dev-docker/fiftyone-app
+        tag: v1.6.0.dev7
+    teamsAppSettings:
+      image:
+        repository: us-central1-docker.pkg.dev/computer-vision-team/dev-docker/fiftyone-teams-app
+        # Note: the naming convention for the image `fiftyone-teams-app` differs from the other images `fiftyone-app`, `fiftyone-app` and `fiftyone-teams-api`
+        # the others are `v1.6.0.dev7` (not `.dev7` vs `-dev7`)
+        tag: v1.6.0-dev.7
+    ```
+
+    > _Note:_ To see the available tags for each image, see
+    > [https://console.cloud.google.com/artifacts/docker/computer-vision-team/us-central1/dev-docker?project=computer-vision-team](https://console.cloud.google.com/artifacts/docker/computer-vision-team/us-central1/dev-docker?project=computer-vision-team)
+
+1. Run skaffold
+
+    ```shell
+    skaffold dev
+
+    # Or with the optional flag
+    # skaffold dev --keep-running-on-failure
+    ```
+
+### Accessing the k8s resources
+
+There are two ways to access resources within the minikube cluster:
+
+* Ingress (recommended)
+* Port Forward
+
+#### Ingress
+
+See
+[Using `minikube tunnel`](https://minikube.sigs.k8s.io/docs/handbook/accessing/#using-minikube-tunnel).
+
+1. Enable the minikube addon `ingress`
+
+    ```shell
+    minikube addons enable ingress
+    ```
+
+1. Install the app via skaffold (see above)
+1. Start the minikube tunnel (and provide sudo password when prompted)
+
+    ```shell
+    $ minikube tunnel
+    ✅  Tunnel successfully started
+
+    📌  NOTE: Please do not close this terminal as this process must stay alive for the tunnel to be accessible ...
+
+    ❗  The service/ingress fiftyone-teams-fiftyone-teams-app requires privileged ports to be exposed: [80 443]
+    🔑  sudo permission will be asked for it.
+    🏃  Starting tunnel for service fiftyone-teams-fiftyone-teams-app.
+    Password:
+    ```
+
+1. Add an entry in to your `/etc/hosts` file to resolve the DNS name
+   `fiftyone.internal` to resolve to localhost
+
+    ```txt
+    127.0.0.1       fiftyone.internal
+    ```
+
+#### Ingress - Login
+
+1. In a web browser and navigate to
+
+    1. Select "Continue with Voxel51 Internal"
+
+1. In a web browser, navigate to
+   [https://fiftyone.internal](https://fiftyone.internal)
+1. Login with `Continue with Voxel51 Internal`
+1. After authentication, you will be redirected to
+   [https://fiftyone.internal/datasets](https://fiftyone.internal/datasets)
+
+> _Note:_ For local development, we use the Auth0 Tenant `dev-fiftyone` and
+> the Auth0 Application `local-dev` contains the setting Allowed Callback URLs
+> (aka Redirect URLs) with [https://fiftyone.internal](https://fiftyone.internal).
+> In `skaffold.yaml` we omit `APP_USE_HTTPS=false`
+> to allow the app to set the Redirect URL protocol to `https`.
```

Comment:
I feel like this is backwards, and [this](https://github.com/voxel51/voxel-hub/blob/main/app/packages/app/pages/api/auth/%5B...auth0%5D.tsx#L19) seems to support my understanding...

`APP_USE_HTTPS=false` should set the redirect protocol to `http` - but if we have a certificate, why are we using `http`?

Reply:
By omitting `APP_USE_HTTPS=false`, we use the app's default which is set to `APP_USE_HTTPS=true`.  (Which is the behavior we want when using TLS certs).

To avoid confusion, I'll rewrite this section to make it clear.